docs/publishing.md

```diff
@@ -0,0 +1,48 @@
+## Versioning
+
+EDS uses [SemVer](https://semver.org/) semantic versioning to keep track of ongoing changes to the product. The three types of versions are:
+
+- **Major (X.y.z)** - Major versions contain breaking changes to Education developers' builds.
+- **Minor (x.Y.z)** - Minor versions add new features or deprecate existing features without breaking changes.
+- **Patch (x.y.Z)** - Patch versions fix defects or optimize existing features without breaking changes.
+
+Look to [this helpful document](http://designsystem.morningstar.com/about/versioning.html) from the Morning Star design system for detailed guidance on versioning.
+
+**Note**: We are currently using a _modified_ form of semver where:
+
+- Breaking changes update the _minor_ version
+- All other changes (new features, fixes, etc.) update the _patch_ version
+
+Once we publish major version 1, we will begin following conventional semver.
+
+---
+
+### Releasing a new version of EDS
+
+_Before the first time you publish_, make sure to:
+
+- set up Two Factor Authentication for your npm account
+- run `npm login` in your terminal to generate an access token for publishing
+
+#### Publishing steps
+
+1. Confirm that all checks are green on CI.
+2. Run `git checkout main`.
+3. Run `yarn` to ensure all dependencies are upto date.
+4. Run `yarn build` to build EDS.
```

Comment:
Should we just have `yarn build` in the release scripts?

Reply:
yeah that's a good idea 👍  `lerna` might have been doing that automatically